client/src/core/client/stream/tabs/Comments/Stream/CommentForm/MediaField.tsx

```diff
@@ -41,7 +41,7 @@ interface Props {
 
 interface Media {
   id?: string;
-  type: "giphy" | "tenor" | "twitter" | "youtube" | "external";
+  type: "giphy" | "tenor" | "twitter" | "bsky" | "youtube" | "external";
```

Comment:
I would suggest that it just stay "bluesky" as the type for this for consistency/clarity, but since it's already set up this way, not sure it's worth changing.

Reply:
nah, I was trying to copy the short form from their domain to be fun... I can find/replace it to "bluesky"

Reply:
fixed